docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -97,6 +102,7 @@ Control the monitor's download speeds, upload speeds, and latency to simulate yo
 Control whether or not to capture screenshots. Options include `'on'`, `'off'`, or `'only-on-failure'`.
 `privateLocations` (`Array<string>`)::
 The <<private-locations,private locations>> to which the monitors will be deployed. These private locations refer to locations hosted and managed by you, whereas 
-<<global-managed-testing-infrastructure,`locations`>> are hosted by Elastic. You can specify a private location using the location's name.
+`locations` are hosted by Elastic. You can specify a private location using the location's name.
```

Comment:
IMO it would be worth informing people where the locations' names can be found. That's because those are not the names in the UI. Instead, they're the names [here](https://github.com/elastic/synthetics/blob/main/src/locations/public-locations.ts).

I imagine it will be frustrating for users if we don't inform these or give examples.

Reply:
@lucasfcosta are you thinking something like this for `locations` and `privateLocations`? 

<img width="865" alt="Screen Shot 2022-10-19 at 3 09 36 PM" src="https://user-images.githubusercontent.com/10479155/196793780-a185656b-b9ef-4bdb-bf5c-fb4f90082183.png">

```asciidoc
`locations` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/locations/public-locations.ts#L28-L37[`Array<SyntheticsLocationsType>`])::
Where to deploy the monitor. Monitors can be deployed in multiple locations so that you can detect differences in availability and response times across those locations.
+
To list available locations you can:
+
* Run the <<elastic-synthetics-locations-command,`elastic-synthetics locations` command>>.
* Go to *{uptime-app}* -> *Monitor Management* and click *Add monitor*.
Locations will be listed in _Monitor locations_.

`privateLocations` (`Array<string>`)::
The <<synthetics-private-location,private locations>> to which the monitors will be deployed. These private locations refer to locations hosted and managed by you, whereas 
`locations` are hosted by Elastic. You can specify a private location using the location's name.
+
To list available private locations you can:
+
* Run the <<elastic-synthetics-locations-command,`elastic-synthetics locations` command>>
with the Kibana URL for the deployment from which to fetch available locations.
* Go to *{uptime-app}* -> *Monitor Management* and click *Add monitor*.
Private locations will be listed in _Monitor locations_ with a "Private" badge next to their names.
```

Reply:
I like the addition of showing the user how to get these with the `locations` command.

It's probably not worth mentioning the UI way of doing things because these names are not exactly the same as what's put into the configuration (e.g. the UI shows `Asia/Pacific - Japan` where as it's defined in the configuration as `japan`.

If we want to add an alternative to `locations`, then a link to https://github.com/elastic/synthetics/blob/main/src/locations/public-locations.ts could be used, but I am not sure this is a friendly to consume as `locations`.

Reply:
I think that's excellent btw! Perhaps a link to the locations file that Paul mentioned would be useful, but I think the instructions you have are sufficient tbh.

Reply:
There's a link to the `public-locations.ts` file on the first line above. 🙂